mono/utils/mono-threads.c

```diff
@@ -828,6 +828,7 @@ mono_thread_info_begin_resume (MonoThreadInfo *info)
 FIXME fix cardtable WB to be out of line and check with the runtime if the target is not the
 WB trampoline. Another option is to encode wb ranges in MonoJitInfo, but that is somewhat hard.
 */
+#ifndef IL2CPP_MONO_DEBUGGER
```

Comment:
Do these removed methods never get called?

Reply:
Yes, they aren't used by any code the debugger needs.